src/BrandEmbassyCodingStandard/Sniffs/Commenting/CreateMockFunctionReturnTypeOrderSniff.php

```diff
@@ -0,0 +1,97 @@
+<?php declare(strict_types = 1);
+
+namespace BrandEmbassyCodingStandard\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function preg_match;
+use const T_FUNCTION;
+
+/**
+ * @final
+ */
+class CreateMockFunctionReturnTypeOrderSniff implements Sniff
+{
+    public const CODE_CREATE_MOCK_FUNCTION_RETURN_TYPE_ORDER = 'CreateMockFunctionReturnTypeOrder';
+
+
+    /**
+     * @return int[]
+     */
+    public function register(): array
+    {
+        return [T_FUNCTION];
+    }
+
+
+    /**
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
+     *
+     * @param int $stackPtr
+     */
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        $functionNamePtr = TokenHelper::findNextEffective($phpcsFile, $stackPtr + 1);
+
+        if ($functionNamePtr === null) {
+            return;
+        }
+
+        $functionName = $tokens[$functionNamePtr]['content'];
+
+        if (preg_match('~^create.*Mock$~', $functionName) !== 1) {
```

Comment:
This is not required name format anymore :man_shrugging: You should decide using return type in `@return` annotation